src/tools/perf/perftest_daemon.c

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright (C) NVIDIA 2023.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "perftest.h"
+
+#include <ucp/api/ucp.h>
+#include <ucs/debug/log.h>
+#include <ucs/sys/string.h>
+#include <ucs/type/serialize.h>
+#include <tools/perf/api/libperf.h>
+#include <string.h>
+#include <arpa/inet.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <signal.h>
+
+typedef struct ucp_perf_daemon_context_t {
+    ucp_context_h                      context;
+    ucp_worker_h                       worker;
+    ucp_listener_h                     listener;
+    uint16_t                           port;
+    ucp_ep_h                           client_ep;
+    ucp_ep_h                           peer_ep;
```

Comment:
Do I understand correctly that `peer_ep` is EP connected to another daemon and `client_ep` is EP connected to host?

If so, I would like to propose to rename `client_ep` to `host_ep`, WDYT?